drizzlepac/hlautils/analyze.py

```diff
@@ -84,7 +83,7 @@ def analyze_data(inputFileList, **kwargs):
 
     fit_method = None  # Fit algorithm used for alignment
     catalog = None     # Astrometric catalog used for alignment
-    catalogSources = 0 # Number of astrometric catalog sources determined based upon coordinate overlap with image WCS
+    catalogSources = 0  # Number of astrometric catalog sources determined based upon coordinate overlap with image WCS
```

Comment:
Variable should not be camelcase. Rename 'catalogSources' to 'catalog_sources' and find/replace the remaining instances.